gpkit/repr_conventions.py

```diff
@@ -14,3 +15,16 @@ def _str(self):
 def _repr_latex_(self):
     "Returns default latex for automatic iPython Notebook rendering."
     return "$$"+self.latex()+"$$"
+
+
+def unitstr(units, into="%s", options="~", dimless=""):
+    "Returns the string corresponding to an object's units."
+    if hasattr(units, "units") and isinstance(units.units, Quantity):
+        units = units.units
+    if not isinstance(units, Quantity):
+        return dimless
+    rawstr = (u"{:%s}" % options).format(units.units)
+    if str(units.units) == "count":
+        rawstr = u"count"  # TODO: remove when pint issue #356 is resolved
+    units = rawstr.replace(" ", "").replace("dimensionless", dimless)
+    return into % units or dimless
```

Comment:
I agree the latter makes more sense, but note that the former is how Python interprets it with no parens (I just tested)